tests/test_errors.py

```diff
@@ -312,6 +314,20 @@ def finalize(self, cfg: Any) -> None:
         ),
         id="dict,accessing_missing_nested_interpolation",
     ),
+    param(
+        Expected(
+            create=lambda: OmegaConf.structured(StructuredInterpolationValidationError),
+            op=lambda cfg: getattr(cfg, "y"),
+            exception_type=InterpolationValidationError,
+            msg=(
+                "While dereferencing interpolation '${.x}': "
+                "Incompatible value 'None' for field of type 'int'"
```

Comment:
Does the error indicate the type containing the problem? (StructuredInterpolationValidationError)?

I would expect `object_type == StructuredInterpolationValidationError`.

Reply:
Yes. Here is the full error message:
```
omegaconf.errors.InterpolationValidationError: While dereferencing interpolation '${.x}': Incompatible value 'None' for field of type 'int'
    full_key: y
    object_type=StructuredInterpolationValidationError
```
To be explicit, I added the `object_type=StructuredInterpolationValidationError` requirement to the `test_errors.py` test-case in https://github.com/omry/omegaconf/pull/754/commits/094a3ee37faadfca0faace720fae1ef13bbaceaa.